buildSrc/src/main/groovy/com/google/firebase/gradle/plugins/FirebaseLibraryPlugin.java

```diff
@@ -64,6 +69,21 @@ public void apply(Project project) {
                 }
               });
     }
+    if(System.getenv().containsKey("METALAVA_BINARY_PATH")) {
+      String metalavaBinaryPath = System.getenv().get("METALAVA_BINARY_PATH");
+      String sourcePath = project.getProjectDir() + "/src/main/java";
```

Comment:
probably needs different paths if its a kotlin project

Reply:
This needs to be replaced with the variant-specific sourceSets collection: see https://android.googlesource.com/platform/frameworks/support/+/refs/heads/androidx-master-dev/buildSrc/src/main/kotlin/androidx/build/java/JavaCompileInputs.kt#43